tests/Microsoft.Identity.Client.Performance/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+
+namespace Microsoft.Identity.Test.Performance
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            BenchmarkRunner.Run<GetAllAccessTokensTests>(
```

Comment:
Looking forward to understanding better what the Benchmark project does.

Reply:
Done. Will add info here: https://aka.ms/msal-net-performance-testing.